apps/www/src/lib/registry/new-york/ui/chart/ChartSingleTooltip.vue

```diff
@@ -43,7 +43,7 @@ function template(d: any, i: number, elements: (HTMLElement | SVGElement)[]) {
     }
     else {
       const style = getComputedStyle(elements[i])
-      const omittedData = [{ name: data.name, value: props.valueFormatter(data[props.index]), color: style.fill }]
+      const omittedData = [{ name: Object.values(data)[0], value: props.valueFormatter(data[props.index]), color: style.fill }]
```

Comment:
The `items` props is expecting type `BulletLegendItemInterface[]`, which has name as the primary identifier. 
I can't load your codesandbox and not sure what issue you are running into.


```
interface BulletLegendItemInterface {
    name: string | number;
    color?: string;
    className?: string;
    shape?: BulletShape;
    inactive?: boolean;
    hidden?: boolean;
    pointer?: boolean;
}
```




https://codesandbox.io/p/devbox/shadcn-vue-donutchartdemo-forked-8z4mt4

Reply:
Hi, I've granted permission to the codesandbox project :)